omero/sysadmins/version-requirements.txt

```diff
@@ -256,62 +168,54 @@ Microsoft Windows
       - Release date
       - (mainline)
       - (extended)
-      - OMERO 5.1
       - OMERO 5.2
       - OMERO 5.3
+      - OMERO 5.4
       - Details
-    * - Vista
-      - from Jan 2007
-      - to Apr 2012
-      - to Apr 2017
-      - Dropped
-      - Dropped
-      - Dropped
-      - `Ref <https://support.microsoft.com/en-gb/help/13853>`__
     * - Win 7
       - from Oct 2009
       - to Jan 2015
       - to Jan 2020
-      - Recommended
       - Supported
       - Unsupported†
-      - `Ref <https://support.microsoft.com/en-gb/help/13853>`__
+      - Unsupported†
```

Comment:
I think we need to remove the lists all together since from what I gather 
those tables were there for server deployment
I don't think it is good to start mixing client recommendation and server recommendation in this page